docs/architecture/incremental.md

```diff
@@ -84,7 +84,17 @@ Some sources cannot define the cursor without user input. For example, in the [p
 
 \(You can find a more technical details about the configuration data model [here](catalog.md)\).
 
+## Getting the Latest Snapshot of data
+
+As demonstrated in the examples above, with **Incremental Append,** a record which was updated in the source will be appended to the destination rather than updated in-place. This means that if data in the source uses a primary key \(e.g: `user_id` in the `users` table\), then the destination will end up having multiple records with the same primary key value. 
+
+However, some use cases require only the latest snapshot of the data. If you want the latest snapshot and are syncing to a destination that supports views, we recommend creating a view on your data which groups by the primary key and deduplicates by the largest `_airbyte_emitted_at` values. The `_airbyte_emitted_at`  column is added by Airbyte to all records synced to the destination. 
+
+{% hint style="info" %}
+Note that in **Incremental Append,** the size of the data in your warehouse increases monotonically since an updated record in the source is appended to the destination rather than updated in-place. If you only care about having the latest snapshot of your data, you may want to periodically run "vacuum" jobs which retain only the latest instance of each record, deduping by primary key. 
```

Comment:
i don't think this is technically a correct use of the concept of database vacuum. maybe just call them clean up jobs?not a big deal if you want to keep it, but i read that and it stood out as not quite accurate.